resources/configurations/examples/change_domain.yaml

```diff
@@ -5,6 +5,11 @@
 # Yaml configuration file version
 version: v2.0
 
+builtin-topics:
+  - name: helloworld_raul
+    type: HelloWorld
+    reliable: true
```

Comment:
I think this should not be here.
I though we would not document reliable as an external option

Reply:
Yes, we agree on that